CHANGELOG.md

```diff
@@ -32,6 +32,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   ([#663](https://github.com/nix-rust/nix/pull/663))
 
 ### Changed
+- On FreeBSD, bind directly to `pipe2` instead of emulating it.  Users should
```

Comment:
Since we tell the user they'll notice no difference, I think it'd be good to say why we're doing this. I'm not certain why we were emulating this in the first place, so that could be clarified here.

Also, this should be updated for NetBSD and OpenBSD also.